migrations/post_install/post_install.sql

```diff
@@ -121,23 +124,29 @@ do $$
       update tjsm T1 set "lastFailedJobId" = T2.id from tjsm T2 where T1."contextTypeCode" = T2."contextTypeCode" and T1."contextValue" = T2."contextValue" and T1.pid = T2.pid and T2."statusCode" = 4003 and T1.rid = T2.rid;
 
       -- creating a temp table temptjsm which have only latest records
-      create temp table temptjsm as WITH cte AS
-      (
-         SELECT id,pid,rid, "createdAt", "contextTypeCode", "contextValue", "contextDetail", "statusCode", "subscriptionId", "createdBy", "updatedBy", "updatedAt", "lastSuccessfulJobId", "lastFailedJobId", ROW_NUMBER() OVER (PARTITION BY "pid", "contextTypeCode", "contextValue" ORDER BY "createdAt" desc)
-      AS rn FROM tjsm
+      create temp table temptjsm as WITH cte AS (
+        SELECT id, pid, rid, "createdAt", "contextTypeCode", "contextValue", "contextDetail", "statusCode", "subscriptionId", "createdBy", "updatedBy", "updatedAt", "lastSuccessfulJobId", "lastFailedJobId",
+          ROW_NUMBER() OVER (PARTITION BY "rid", "contextTypeCode", "contextValue" ORDER BY "createdAt" desc)
```

Comment:
Updated partitioning by resourceId from projectId, since we need to group by resourceIds